feature/gribi/otg_tests/get_rpc_test/get_rpc_test.go

```diff
@@ -345,10 +356,20 @@ func testIPv4LeaderActive(ctx context.Context, t *testing.T, args *testArgs) {
 	gnmi.Update(t, args.dut, niProto.Config(), static)
 	validateGetRPC(ctx, t, args.clientA)
 	for ip := range ateDstNetCIDR {
-		ipv4Path := gnmi.OC().NetworkInstance(deviations.DefaultNetworkInstance(args.dut)).Afts().Ipv4Entry(ateDstNetCIDR[ip])
-		if got, want := gnmi.Get(t, args.dut, ipv4Path.State()).GetPrefix(), ateDstNetCIDR[ip]; got != want {
-			t.Errorf("ipv4-entry/state/prefix got %s, want %s", got, want)
-		}
+	        ipv4Path := gnmi.OC().NetworkInstance(deviations.DefaultNetworkInstance(args.dut)).Afts().Ipv4Entry(ateDstNetCIDR[ip])
+                lastValue, ok := gnmi.Watch(t, args.dut, ipv4Path.State(), time.Minute, func(v *ygnmi.Value[*oc.NetworkInstance_Afts_Ipv4Entry]) bool {
+                        return v.IsPresent()
+                }).Await(t)
+
+                if ok {
```

Comment:
more idiomatic to check the error condition ("if !ok") first and then the non-error case doesn't need to be in "else"